src/views/v2/month/mobile-events/mobile-day.php

```diff
@@ -49,13 +51,19 @@
 ?>
 
 <div <?php tribe_classes( $classes ); ?> id="<?php echo sanitize_html_class( $mobile_day_id ); ?>">
-	<?php $this->template( 'month/mobile-events/mobile-day/day-marker', [ 'day_date' => $day_date ] ); ?>
 
-	<?php foreach( $events as $event ) : ?>
+	<?php foreach ( $events as $event ) : ?>
+		<?php $event_date = $event->dates->start->format( Dates::DBDATEFORMAT ); ?>
+
+		<?php if ( empty( $running_date ) || $running_date !== $event_date ) : ?>
+			<?php $this->template( 'month/mobile-events/mobile-day/day-marker', [ 'day_date' => $event_date ] ); ?>
+		<?php endif; ?>
+
 		<?php $this->setup_postdata( $event ); ?>
 
 		<?php $this->template( 'month/mobile-events/mobile-day/mobile-event', [ 'event' => $event ] ); ?>
 
+		<?php $running_date = $event_date; ?>
```

Comment:
Would love to see these changes be added via some sort of extra template included into before `month/mobile-events/mobile-day/mobile-event`.

Reply:
Not sure I agree - unless we specifically don't want this to happen on "normal" month view?

Reply:
There is one caveat around multiday events.
current: https://d.pr/i/ahWMmH
this change: https://d.pr/i/xZRGME
...as the event date is always the start day.

Reply:
see `src/views/v2/list/month-separator.php`, we use `Tribe\Events\Views\V2\Utils\Separators` to do this sort of logic.

Reply:
Done. See https://github.com/the-events-calendar/the-events-calendar/pull/3485/files#diff-f745177a36763b9a049d9d3973bb84e45bc46ffb6849a399c231da9aaf6b7703